_pytest/nodes.py

```diff
@@ -178,15 +179,23 @@ def add_marker(self, marker):
         elif not isinstance(marker, MarkDecorator):
             raise ValueError("is not a string or pytest.mark.* Marker")
         self.keywords[marker.name] = marker
+        self._markers.update([marker])
+
+    def find_markers(self, name):
```

Comment:
Does this automatically make it to the reference docs because the whole class is autodoc'd?

Reply:
i beleive so

im planning to work on the docs a bit more